inventory/host_vars/prod2.openfoodnetwork.org.au/config.yml

```diff
@@ -9,18 +9,17 @@ mail_domain: openfoodnetwork.org
 
 # Optional list of subdomains to register in letsencrypt certificate
 # Defaults to {{ domain }} if list is undefined
-#
-#certbot_domains:
-#  - openfoodnetwork.org.au
-#  - www.openfoodnetwork.org.au
-#  - prod2.openfoodnetwork.org.au
+certbot_domains:
+  - openfoodnetwork.org.au
+  - www.openfoodnetwork.org.au
+  - prod2.openfoodnetwork.org.au
```

Comment:
has this ever existed? I mean, are there links out there in the wild that we might need to maintain?

Reply:
It has existed during migration to the new server. But there shouldn't be any links to this out there. I can be handy for the next migration to a new server though. While domains are changing and nameservers deliver differing responses, we can use nginx's proxy to point to the right server.